verilog/tools/kythe/indexing_facts_tree_extractor_test.cc

```diff
@@ -410,6 +410,61 @@ TEST(FactsTreeExtractor, ModuleWithPortsTest) {
   EXPECT_EQ(result_pair.right, nullptr) << *result_pair.right;
 }
 
+TEST(FactsTreeExtractor, ModuleWithPortsNonANSIStyleTest) {
+  constexpr int kTag = 1;  // value doesn't matter
+  const verible::SyntaxTreeSearchTestCase kTestCase = {{"module ",
+                                                        {kTag, "foo"},
+                                                        "(",
+                                                        {kTag, "a"},
+                                                        ", ",
+                                                        {kTag, "b"},
+                                                        ");\nendmodule: ",
```

Comment:
suggest comment:
// Normally, tools will reject non-ANSI port declarations that are missing their full definitions inside the body like "input a", but here we don't care and are just checking for references, even if they are dangling.